<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="C4LAP149" ignored="0" name="TC_26_Company_Internet_with_Wlan_with_Company_Voice_Automatic" tests="1" failures="1" timestamp="11 Jan 2018 06:43:16 GMT" time="764.219" errors="0">
  <testcase name="TestCaseExecution" time="700.289" classname="TestDriver.TestExecutionDriver">
    <failure type="java.lang.AssertionError" message="expected [Ready for Billing] but found [Requested]">
      <![CDATA[java.lang.AssertionError: expected [Ready for Billing] but found [Requested]
	at com.test.automation.OM_Automation.uiActions.FinalValidationOfOrderCompletion.orderStatusCheck(FinalValidationOfOrderCompletion.java:148)
	at com.test.automation.OM_Automation.uiActions.SubOrders.CreatedOrderValidationGeneric(SubOrders.java:185)
	at DUMMYDELETEIT.genericDriver.subOrdersValidationAndCompleteOrderInConsole(genericDriver.java:669)
	at TestDriver.TestExecutionDriver.TestCaseExecution(TestExecutionDriver.java:190)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 22 stack frames]]>
    </failure>
  </testcase> <!-- TestCaseExecution -->
</testsuite> <!-- TC_26_Company_Internet_with_Wlan_with_Company_Voice_Automatic -->
